src/providers/search/__mocks__/search-results.mock.ts

```diff
@@ -6,8 +6,8 @@ export const searchResultsMock: SearchResultTestSchema [] = [
     applicationReference: '123456',
     category: 'B',
     costCode: 'abcd',
-    staffNumber: '98765',
```

Comment:
Why is `staffNumber` no longer required?

Reply:
it's not a field we display on the search results page so aren't allowed to pass it back